backend/src/routes/api/gpu/gpuUtils.ts

```diff
@@ -62,10 +62,32 @@ export const getGPUNumber = async (fastify: KubeFastifyInstance): Promise<GPUInf
     areGpusConfigured = true;
   }
 
+  // const data: GPUInfo = {
+  //   configured: areGpusConfigured,
+  //   available: maxGpuNumber,
+  //   autoscalers: scalingLimit,
+  // };
   const data: GPUInfo = {
-    configured: areGpusConfigured,
-    available: maxGpuNumber,
-    autoscalers: scalingLimit,
+    configured: true,
+    available: 4,
```

Comment:
Is this intended to be merged, it seems that these values are fixed, I have little context but it seems kind weird.

Reply:
yeah good catch. this was used to test gpus before i had a cluster (hard coding the backend). I have updated it